### ISSUE TITLE
Method "init_by_array" is never used.

### ISSUE THREAD

**Issue by virtyaluk** [chancejs/chancejs#254]
Hey!

I've noticed that [init_by_array](https://github.com/chancejs/chancejs/blob/master/chance.js#L3082) method is not used at all.
What's the purpose of keeping that method with such a complex logic?

**Comment by virtyaluk** [comment]
There is also few other not used methods related to the Mersenne Twister. I believe removing them will reduce the size of package for a little.

**Comment by victorquinn** [comment]
Good call! 

The Mersenne Twister was just copy/pasted in from another open source project hence its unused vestigial bits like that.

**Comment by virtyaluk** [comment]
Yeah, I've got it. I'm just saying that unused parts may be deleted to reduce the size of the package.

I leave this for your consideration.

While the issue is not really an issue you may close it.
